src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/dto/VedtaksperiodeDto.kt

```diff
@@ -6,8 +6,10 @@ import no.nav.tilleggsstonader.sak.vedtak.domain.Vedtaksperiode
 import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.AktivitetType
 import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.MålgruppeType
 import java.time.LocalDate
+import java.util.UUID
 
 data class VedtaksperiodeDto(
+    val id: UUID = UUID.randomUUID(),
```

Comment:
Hadde nog vært fint hvis frontend ansvarer for å sende inn id på vedtaksperiode som i 
* https://github.com/navikt/tilleggsstonader-sak-frontend/pull/691